### ISSUE TITLE
[Bug] 

### ISSUE THREAD

**Issue by m1ntyduck** [IceWhaleTech/CasaOS#1143]
**Describe the bug**
When using proxmox / debian 11 / lxc any network shares created, are accessible with any security 'guest access with full read and wrote permisions

**To Reproduce**
1. Install Debian LCX
2. Install Casa OS
3. Use file manager to created SMB
4. SMB crated
5. Access SMB share from other devices - no password requirement

**Expected behavior**
Expect to have to authenticate with CasaOS - username and password

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Proxmox 7 4-3
 - Browser Chrome
 - Version 113.0.5672.126

**Comment by Vizion-HUN** [comment]
Hi, same error here. It would be nice to have a "Shares" menu, to add users, set up permissions. And at least use smb2 or better...
I did not find info about smb on host. What happends if I set up shares on my ubuntu server host, and not in Casa OS UI?
Worth a try...

**Comment by lewgrant** [comment]
Shares are a no-go if anyone on the network gets full read-write no matter what.
Until the native share feature in the files app can be enhanced to allow setting username/passwords is there an app in the app store that can be recommended for creating and managing SMB/NFS shares?

**Comment by Triskae** [comment]
Yep, It would be so nice to have some kind of menu to manage the users !

**Comment by plangin** [comment]
+1

Open the shares to any guest is just a security flaw.

Until then I use [crazymax/samba](https://github.com/crazy-max/docker-samba)

**Comment by inkly** [comment]
This is addressed in the inkly distribution of CasaOS, v0.4.40.

A share can be restricted to an account, and accounts are managed from the Shared Folders screen. They are separate from the CasaOS login on purpose: Samba keeps its own password database, so reusing the web credentials would mean storing the password a second time in a recoverable form. The accounts have no shell and cannot log in to the host. An existing share can be switched from guest to account access without recreating it.

The generated config also drops `force user = root` and the 0777 masks, which made any authentication decorative.

    curl -fsSL https://github.com/inkly/CasaOS-Install/releases/latest/download/install.sh | sudo bash

Details: https://github.com/inkly/CasaOS-Install#what-is-in-v0440 — upstream has not shipped since 2025; this is a maintained release, not a patch to it.
